test/export/test_export_training_ir_to_run_decomp_non_strict.py

```diff
@@ -0,0 +1,52 @@
+# Owner(s): ["oncall: export"]
```

Comment:
Maybe this should just be merged into the strict training IR file? That would run both strict/non-strict at once

Reply:
sounds good to me